packages/@angular/cli/commands/eject.ts

```diff
@@ -0,0 +1,61 @@
+import { BuildOptions } from '../models/build-options';
+import { Version } from '../upgrade/version';
+
+const Command = require('../ember-cli/lib/models/command');
+
+// defaults for BuildOptions
+export const baseEjectCommandOptions: any = [
```

Comment:
These should be referenced from `baseBuildCommandOptions` in `commands/build.ts`.